File: Python programming/ArmStrong.py
n=153
n1=n
n2=n
count=0
sum=0

while n>0:
     count+=1
     n//=10


while n1>0:
     digit=n1%10
     pow=1

     for i in (1,count+1):
          pow=pow*digit
     
     print(pow)
     sum=sum+pow
     n1//=10

if n2==sum:
     print("arm strong number")
else:
     print("not arm strong")

File: java programming/ArmStrong.java
public class ArmStrong {
    public static void main(String[] args) {
        
        int n=153;
        int n1=n;
        int n2=n;
        int count=0;
        int sum=0;

        //calculate the count of digit
        while(n>0){
            count++;
            n/=10;
        }

        //find the power and sum
        while(n1>0){
            int digit=n1%10;
            int pow=1;

            for(int i=1;i<=count;i++){
                pow=pow*digit;
            }
            sum=sum+pow;
            n1=n1/10;
        }

        if(n2==sum) System.out.println("arm strong");
        else System.out.println("not arm strong");
    }
}
